src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Kindly use the ducks' pattern here as well

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
  ```

Reply:
Resolved